lib/src/main/java/com/diffplug/spotless/Formatter.java

```diff
@@ -103,6 +112,8 @@ public static Formatter.Builder builder() {
 	}
 
 	public static class Builder {
+		// optional parameters
+		private String name = "misc";
```

Comment:
I would call this "unnamed formatter" rather "misc".

Reply:
I suggest _unnamed_ as _unnamed formatter_ looks a bit redundant.

Reply:
Sounds good.